azure-keyvault-webkey/src/main/java/com/microsoft/azure/keyvault/webkey/JsonWebKey.java

```diff
@@ -46,32 +63,38 @@
     /**
      * Key Identifier.
      */
+	@JsonProperty(value = "kid")
     private String kid;
 
     /**
-     * Key type, usually RSA. Possible values include: 'EC', 'RSA', 'RSA-HSM',
-     * 'oct'.
+     * JsonWebKey key type (kty). Possible values include: 'EC', 'EC-HSM',
+     * 'RSA', 'RSA-HSM', 'oct'.
      */
+    @JsonProperty(value = "kty")
     private JsonWebKeyType kty;
 
     /**
      * The keyOps property.
      */
+    @JsonProperty(value = "key_ops")
     private List<JsonWebKeyOperation> keyOps;
 
     /**
      * RSA modulus.
      */
+    @JsonProperty(value = "n")
     private byte[] n;
 
     /**
      * RSA public exponent.
      */
+    @JsonProperty(value = "e")
     private byte[] e;
 
     /**
-     * RSA private exponent.
+     * RSA private exponent, or the D component of an EC private key.
      */
+    @JsonProperty(value = "d")
     private byte[] d;
```

Comment:
why are some of the properties marked with the @JsonProperty attribute, but not all?

Reply:
I just left it as implemented (with some with, some without) - will go ahead and add them all in.